### DISCUSSION TITLE
Code splitting causes PSI drop with 9.2+

### DISCUSSION THREAD

**Discussion by mintyfloss** [vercel/next.js#14940] [Help]
# Feature request

## Is your feature request related to a problem? Please describe.

We're currently pinned to version 9.1.7 because upgrading to a more recent version causes our PSI to drop significantly. Here are the screenshots of the Lighthouse result.

||**9.1.7**| **9.4.4**|
|----|-----|-----|
|**Lighthouse**|<img width="957" alt="Screen Shot 2020-07-07 at 2 25 19 PM" src="https://user-images.githubusercontent.com/885453/86832458-d0745d80-c05d-11ea-828d-001b4741e823.png">|<img width="961" alt="Screen Shot 2020-07-07 at 2 19 41 PM" src="https://user-images.githubusercontent.com/885453/86832513-dcf8b600-c05d-11ea-960b-16f0489e75a9.png">|
|**JS Chunks**|5 JS files<br /><img width="220" alt="Screen Shot 2020-07-07 at 2 21 51 PM" src="https://user-images.githubusercontent.com/885453/86832886-64462980-c05e-11ea-85ab-510fced2a84c.png">|20 JS files<br /><img width="218" alt="Screen Shot 2020-07-07 at 2 35 13 PM" src="https://user-images.githubusercontent.com/885453/86833408-1b42a500-c05f-11ea-9291-7c2330f9506a.png">|
|**First Contentful Paint**|2.3s|4.8s|

## Describe the solution you'd like

I'm guessing that in version 9.4.4, PSI is suffering because it's loading an additional 15 JS files. Is there a way to customize code-splitting? We have 12 different pages on our site, but most of our users rarely navigate between pages. At most, users will go to 2 pages and leave the site.

I'm also curious how the common chunk is created. We include the `react-ga` component in `_app.js`, so I expect the code for it will be in `common`. However, I see a separate chunk created for it.

## Describe alternatives you've considered

I currently can't think of another reason why our PSI score will suffer. There was no code change besides updating to Next to 9.4.4.

I also posted this same issue in this thread https://github.com/vercel/next.js/discussions/11120#discussioncomment-28216.

**Comment by jamesmosier** [comment]
Here is more info on the new code splitting techniques: https://nextjs.org/blog/next-9-2#improved-code-splitting-strategy

And additionally here is the webpack splitChunks config:

https://github.com/vercel/next.js/blob/38bd1a024cb25923d8ea15f269a7294d073684d8/packages/next/build/webpack-config.ts#L373-L457


Currently there is no configuration options for code splitting exposed.
